src/server/migrations/20190925112652_vessels_MMSI_type_change.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+exports.up = function(knex) {
+  return knex.schema.alterTable("vessels", table => {
+    table
+      .string("MMSI", 9)
```

Comment:
is it okey to use "string" or "varchar", because in the task it is mentioned to change the type from decimal to varchar

Reply:
@fawadik In knex "string" means "varchar". if you run the code and take look at DB you can see the previous type is changed to varchar.

Reply:
yeah...cool Omid..... i did check it...